UserService/src/main/java/com/streamster/userservice/model/Preferences.java

```diff
@@ -0,0 +1,15 @@
+package com.streamster.userservice.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class Preferences {
+
+    private List<String> studyPrograms;
```

Comment:
I am thinking if it's not better to have it as a Set<String>, to avoid having duplicates in dbs

Reply:
actually in tags it would be more relevant

Reply:
ok